src/app/compliance/CompliancePage.js

```diff
@@ -179,6 +196,32 @@ class CompliancePage extends Component<Props, State> {
     this.props.addInvestor()
   }
 
+  handleFreezeModalOpen = () => {
+    // $FlowFixMe
+    this.props.confirm(
+      <div>
+        <p>
+        Once you hit &laquo;CONFIRM&raquo;, the freeze on all transfers will PREVENT ANY INVESTOR FROM BUYING
+        OR SELLING YOUR TOKENS UNTIL YOU RESUME TRANSFERS. Consider notifying all your investors. If you wish
+        to review with your Advisors, please select &laquo;CANCEL&raquo;.
+        </p>
+      </div>,
+      () => {
+        this.props.toggleFreeze((()=>{
+          this.setState({ isFrozenModalOpen: true })
+        }))  
+      },
+      "Pause All Transfers?"
+    )
+  }
+
+  handleUnfreezeConfirm = () =>{
+    this.setState({ isFrozenModalOpen: false })
+    this.props.toggleFreeze()
+    //todo @JoseMiguelHerrera, isFrozenModalOpen should be set to true if the toggleFreeze()
+    //fails, but this would require ui.tx to handle errors, with a "after fail" function
```

Comment:
`//fails` > `// TODO @JoseMiguelHerrera: ...fails`
otherwise it looks like separate comment rather than end of the TODO description, particularly because IDE won't highlight second line as TODO